flexmeasures/data/models/reporting/tests/test_aggregator.py

```diff
@@ -153,3 +154,22 @@ def test_resampling(setup_dummy_data):
     assert result.event_starts[0] == pd.Timestamp(
         year=2023, month=10, day=29, tz="Europe/Amsterdam"
     )
+
+
+def test_multiple_sources(setup_dummy_data):
+    """check that it uses the first source as default, instead of using all of them."""
```

Comment:
Lets's update the docstring to avoid potential future confusion. We now just check that each event has a single belief, right?

Reply:
You're right. Actually, there are multiple events with the same event_start.